app/models/invitation.rb

```diff
@@ -0,0 +1,13 @@
+class Invitation < ActiveRecord::Base
+  belongs_to :inviter, class_name: "User"
+  validates_presence_of :friends_name, :friends_email, :message
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.